.eslintrc.yml

```diff
@@ -0,0 +1,20 @@
+env:
+  browser: true
```

Comment:
sure?

Reply:
no... this was not deliberate. i just used boilerplate code. 
With [WebUSB](https://wicg.github.io/webusb/) we might run in the browser someday... 
But i will delete that line and put it back in comes the time we need this to really run in the browser...